module/API/Measurer.cpp

```diff
@@ -0,0 +1,79 @@
+/**
+ * @file Measurer.cpp
+ * @brief 計測に用いる関数をまとめたラッパークラス
+ * @author YKhm20020
+ */
+
+#include "Measurer.h"
+
+// 明るさを取得
+// 参考: https://tomari.org/main/java/color/ccal.html
+int Measurer::getBrightness()
+{
+  // RGBモードと光センサモードを併用すると動作が悪くなるためRGBモードで取得する
+  // 参考: https://qiita.com/kawanon868/items/5d52eb291c3f71af0419
+  rgb_raw_t rgb = getRawColor();
+  int brightness = std::max({ rgb.r, rgb.g, rgb.b }) * 100 / 255;  // 明度を取得して0-100に正規化
+  return brightness;
+}
+
+// RGB値を取得
+rgb_raw_t Measurer::getRawColor()
+{
+  rgb_raw_t rgb;
+  ev3api::ColorSensor(colorSensorPort).getRawColor(rgb);
```

Comment:
もしかして↓で動く？
```suggestion
  Measurer::colorSensor->getRawColor(rgb);
```

以降の「ev3api::〇〇(Measurer::△△Port).xx」も同様に
「Measurer::△△->xx」